Prac_05/emails.py

```diff
@@ -0,0 +1,30 @@
+"""
+CP1404 Practical
+Emails
+"""
+
+EMAILS = {}
+
+
+def main():
+    user_email = input("Email: ")
+    while user_email != "":
+        user_name = user_email.split('@')
+        if '.' in user_name:
+            full_name = user_name[0].split('.')
+            fullname = "{} {}".format(full_name[0], full_name[1]).title()
+        else:
+            fullname = user_name[0].title()
+        name_check = input("Is your name {}? (y/n) ".format(fullname)).lower()
+        if name_check == 'n':
```

Comment:
its fine to just have check for "n" as input but consider expecting other input as well, in this case "no"?